pysph/examples/taylor_green.py

```diff
@@ -259,14 +260,14 @@ def create_particles(self):
         fluid.h[:] = self.hdx * dx
 
         corr = self.kernel_correction
-        if corr == 'mixed':
+        if corr in ['mixed', 'crksph']:
             fluid.add_property('cwij')
```

Comment:
can't just we add 'cwij' as a 9 element array to the precomputed variables, so this could be generalized.

Reply:
For now I think this is fine, this needs to be considered carefully.